libnavigation-core/src/main/java/com/mapbox/navigation/core/adasis/ProfileLong.kt

```diff
@@ -0,0 +1,55 @@
+package com.mapbox.navigation.core.adasis
+
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+
+/**
+ * Profile long message options
+ *
+ * @param options common options for profile long message
+ * @param types options for each type of profile long message
+ */
+@ExperimentalPreviewMapboxNavigationAPI
+class ProfileLong(
```

Comment:
internal constructor + builder
Otherwise it'll be more difficult to extend the API.
But probably also not critical for the snapshot.